app/src/main/java/com/example/mspaint/mainui/mainScreen.kt

```diff
@@ -94,7 +88,7 @@ fun MainScreen() {
                                             lines.add(line)
                                         }
                                     }
-                                ) // creates the actual drawing. drawLine function is a part of Jetpack Compose
```

Comment:
I hope to remain all comment like this that explain what these parts of code doing so I can edit them without any worries.